src/Stratis.Features.FederatedPeg/TargetChain/CrossChainTransferStore.cs

```diff
@@ -997,37 +1000,37 @@ private OutPoint EarliestOutput(Transaction transaction)
 
         private ICrossChainTransfer[] GetTransfersByStatusInternal(CrossChainTransferStatus[] statuses, bool sort = false, bool validate = true)
         {
-            lock (this.lockObj)
-            {
-                this.Synchronize();
-
-                var depositIds = new HashSet<uint256>();
-                foreach (CrossChainTransferStatus status in statuses)
-                    depositIds.UnionWith(this.depositsIdsByStatus[status]);
-
-                uint256[] partialTransferHashes = depositIds.ToArray();
-                ICrossChainTransfer[] partialTransfers = this.Get(partialTransferHashes).Where(t => t != null).ToArray();
+            var depositIds = new HashSet<uint256>();
```

Comment:
Can you be consistent and end the method name with Locked `GetTransfersByStatusInternalLocked` which means the method must be called form a method that has the lock.

Reply:
The suffix is redundant as all private methods are either called by a public method with a lock or by another private method.

Reply:
Done.